CONTRIBUTING.md

```diff
@@ -83,3 +83,25 @@ These docstrings can be fairly simple, but can also make use of [reStructuredTex
 ## Infrastructure & configuration files
 
 TODO
+
+
+### Visual Studio Code Development Environment
```

Comment:
These are good notes, but very specific for one particular IDE. Are we going to add notes for every IDE?

Reply:
To be honest I mostly added them in case I end up having to get back into it after having done something else for a couple of months. I have updated the comment to indicate that text editors are fine and there are many IDEs

Reply:
Haha, I was just worried it would get too long / out-of-scope if everyone started adding their own editor setups :D